source/brailleDisplayDrivers/albatross/driver.py

```diff
@@ -8,12 +8,14 @@
 for description of most important functions.
 """
 
+import ftdi2

```

Comment:
this library is incredibly old and not available online any more. We should avoid adding new dependencies on it.
Instead, use the pip module `pyftdi` and add it to `requirements.txt`.

Reply:
from [installation](https://eblot.github.io/pyftdi/installation.html):

"Windows is not officially supported (i.e. not tested) but some users have reported successful installations. Windows requires a specific libusb backend installation."

`from pyftdi.ftdi import Ftdi
Ftdi.show_devices()`
fails.

Pyusb is installed, and libusb-1.0.dll is copied (32-bit c:\windows\syswow64, 64-bit c:\windows\system32).

Reply:
We should probably use Leonard's brlapi build of libusb as it should support py3.11, I'm not sure if the dll you are using supports 3.11

However since windows support is not known, it is probably riskier to move to this library.

This is unfortunate as our current module is unmaintained as well, and likely to break at some point in the future.

Reply:
Can you @leonardder give some advice how I might get libusb to work? I have copied dlls from libusb-1.0.26-binaries.7z.